third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json

```diff
@@ -81,12 +81,13 @@
             {
                 "key": "frequency",
                 "displayName": "(Optional) Frequency",
-                "displayHint": "Frequency in minutes",
+                "displayHint": "Frequency in minutes, default: 1",
```

Comment:
I don't think 1 is a great default. Although I just checked and HTTP monitors also have a default of 1min while browser monitors default to 15min. 5 or 15 seems more reasonable to me.
What do you think @pkastner ?

Reply:
Let's go with 15min as default.

Reply:
@dlopes7 and you just wanted to improve the hint. Sorry ;) :D